mne/evoked.py

```diff
@@ -182,6 +181,12 @@ def __init__(self, fname, condition=None, baseline=None, proj=True,
                 elif my_kind == FIFF.FIFF_CH_INFO:
                     tag = read_tag(fid, pos)
                     chs.append(tag.data)
+                elif my_kind == FIFF.FIFF_FIRST_TIME:
+                    tag = read_tag(fid, pos)
+                    first_time = float(tag.data)
+                elif my_kind == FIFF.FIFF_NO_SAMPLES:
+                    tag = read_tag(fid, pos)
+                    nsamp = int(tag.data)
 
```

Comment:
There isn't really any magic. There are four things happening in this PR that we needed:

1. Recognize that the file looks like an evoked file does on disk.
2. Add a couple of missing FIFF constants.
3. Enhance the plotting and pick functions to deal with new constants.
4. Fix the evoked reading code to deal with a different way of defining the start and end points -- we usually use `first` and `last`, here we have `first_time` (with `sfreq`) and `no_samples` to get the same information.

This particular point in the code is where 4) happens.